mecanum_drive_controller/src/mecanum_drive_controller.yaml

```diff
@@ -0,0 +1,121 @@
+mecanum_drive_controller:
+  reference_timeout: {
+    type: double,
+    default_value: 0.0,
+    description: "Timeout for controller references after which they will be reset. This is especially useful for controllers that can cause unwanted and dangerous behavior if reference is not reset, e.g., velocity controllers. If value is 0 the reference is reset after each run.",
+
+  }
+
+  joint_names: {
+    type: string_array,
+    default_value: [],
+    description: "Name of the wheels joints.",
+    read_only: false,
+
+  }
+
+  state_joint_names: {
```

Comment:
Maybe it would be better to actually use specific `command_joint_names` since this is more relevant for the following controller.